docs/how-to/extend-metrics.md

```diff
@@ -1,4 +1,36 @@
 # Extend Metrics
 
+Sometimes, though, you’ll have the need for a metric that’s not present in the list of node_exporter metrics out of the box. 
+The textfile collector allows exporting of statistics from batch jobs. It can also be used to export static metrics, such as what role a machine has. 
+
 ## Enable the textfile collector
 
+The collector is enabled by default. Next folders are used for different resolutions:
+
+| Resolution | Folder                                                                  |
+|------------|-------------------------------------------------------------------------|
+|  High      | /usr/local/percona/pmm2/collectors/textfile-collector/high-resolution   |
+|  Medium    | /usr/local/percona/pmm2/collectors/textfile-collector/medium-resolution |
+|  Low       | /usr/local/percona/pmm2/collectors/textfile-collector/low-resolution    |
+
+![!image](../_images/node-exporter.textfile-collector.1.png)
+
+Exporter parse all files in that directories matching the glob *.prom using the text format.
+
```

Comment:
```suggestion
Files in the folders are expected to be parsed in [Exposition format](https://prometheus.io/docs/instrumenting/exposition_formats/) and metrics stored on PMM Server-side with an additional label related to this Node.

```

Reply:
I have added the link and the last sentence.

![Screenshot 2021-09-29 at 11 56 42](https://user-images.githubusercontent.com/25184101/135237014-ab10e53e-d0f4-4dd7-b41c-dee32e33b1b3.png)